plugins/gimp/file-jxl-load.cc

```diff
@@ -62,9 +82,9 @@ bool LoadJpegXlImage(const gchar *const filename, gint32 *const image_id) {
 
   auto dec = JxlDecoderMake(nullptr);
   if (JXL_DEC_SUCCESS !=
-      JxlDecoderSubscribeEvents(dec.get(),
-                                JXL_DEC_BASIC_INFO | JXL_DEC_COLOR_ENCODING |
-                                    JXL_DEC_FULL_IMAGE | JXL_DEC_FRAME)) {
```

Comment:
It seems you need DEC_FRAME below, so it should not be removed from the subscribed events.

Reply:
oops